src/OrbitCaptureClient/include/OrbitCaptureClient/CaptureListener.h

```diff
@@ -20,11 +21,8 @@ class CaptureListener {
   virtual ~CaptureListener() = default;
 
   // Called after capture started but before the first event arrived.
```

Comment:
This comment is now no longer correct, is it?

Reply:
removed.